bigchaindb/commands/bigchaindb.py

```diff
@@ -191,6 +190,36 @@ def run_upsert_validator_approve(args, bigchain):
         return False
 
 
+def run_upsert_validator_show(args, bigchain):
+    """Retrieves information about an upsert-validator election
+
+    :param args: dict
+        args = {
+        'election_id': the transaction_id for an election (str)
+        }
+    :param bigchain: an instance of BigchainDB
+    """
+
+    election = bigchain.get_transaction(args.election_id)
+    if not election:
+        logger.error(f'No election found with election_id {args.election_id}')
+        return
+
+    new_validator = election.asset['data']
+
+    public_key = public_key_to_base64(new_validator['public_key'])
+    power = new_validator['power']
+    node_id = new_validator['node_id']
+    status = election.get_status(bigchain)
+
+    response = f'public_key={public_key}\npower={power}\nnode_id={node_id}\nstatus={status}'
+
+    logger.info(response)
+
+    # TODO: Remove this once tests/commands/test_utils.test_configure_bigchaindb_logging.py is fixed
```

Comment:
We don't actually need to return anything since this is executed from the command line. We just log the info so they see it in `stdout`

The return value is just there to make this testable, since the damn `utils` test is breaking the `logger`

Reply:
My point is, it's perfectly fine to have a return value and use it in tests. Asserting logs, on the other hand is very cumbersome. So, there is no need in these TODOs.

Reply:
Fair point... Gone.